.claude/skills/poll/SKILL.md

```
@@ -22,19 +22,58 @@ allowed-tools:
+| 0 | All checks pass | Ready to merge |
+| 1 | CI failures | Fix code, push, re-run (background again) |
+| 2 | Checks pending | Should not exit 2 unless --max-polls hit; re-run |
+| 3 | Inline review comments | Address comments, push, re-run (background again) |
+| 4 | Awaiting human review | Reviewers notified; CronCreate to re-poll later |
+
+## Exit 4 — schedule a re-poll cron
+
+When the script exits 4, it has already posted a comment and requested reviewers.
+Use CronCreate to check back automatically:
+
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)